src/coreclr/tools/aot/crossgen2/Program.cs

```diff
@@ -330,6 +339,13 @@ private int Run(string[] args)
                 return 1;
             }
 
+            if (_commandLineOptions.Version)
+            {
+                string version = GetCompilerVersion();
+                Console.WriteLine(version);
+                return 1;
```

Comment:
```suggestion
                return 0;
```
That's what other tools return:
```
jkotas@jkotas7:~$ dotnet --version
6.0.201
jkotas@jkotas7:~$ echo $?
0
jkotas@jkotas7:~$ dotnet sos --version
6.0.257301+27172ce4d05e8a3b0ffdefd65f073d40a1b1fe54
jkotas@jkotas7:~$ echo $?
0
```